unleash-yggdrasil/src/lib.rs

```diff
@@ -37,6 +37,7 @@ pub struct CompiledToggle {
     pub variants: Vec<CompiledVariant>,
     pub impression_data: bool,
     pub project: String,
+    pub dependencies: Vec<FeatureDependency>,
```

Comment:
I decided to follow the pattern of variants, where it's always a vec and we convert a `None` from the API into an empty Vec. Happy to go ahead and wrap it in an option if we think that's better.

Reply:
I think this is okay. An Option would have made the actual implementation a little easier but Unleash is so fond of sending empty arrays for nulls that I think this is correct